vmm/src/device_manager.rs

```diff
@@ -4064,7 +4064,8 @@ impl Aml for DeviceManager {
                         #[cfg(target_arch = "aarch64")]
                         &"ARMH0011",
                     ),
-                    &aml::Name::new("_UID".into(), &aml::ZERO),
+                    &aml::Name::new("_UID".into(), &aml::ONE),
+                    &aml::Name::new("_DDN".into(), &"COM1"),
```

Comment:
Can you explain why you needed this change? I'm asking because it's also affecting `x86_64`.

Reply:
Yep, there is a couple considerations behind this.

- _UID change to 1 is a cosmetic change. The device is currently referred as COM1 everywhere.
- _DDN is required as it's referred by some driver code in Windows IoT. There should be no impact to x86 even if this is present. 

Another thought was also based on this note from the ARM Base Boot Requirements doc:

```
If OS debug through serial port is wanted, the system must make the UART instance that is 
specified in DBG2 to be exclusively available for use of the debugger. How the system enables 
this is IMPLEMENTATION DEFINED.
```

We currently only implement one serial port by design, which seems to work. In the Windows guest however, there's also  SAC console enabled on the same device. The debug seems to work ATM, but based on the spec it might need at least two serial devices in the future.

Both changes can be scoped into AARCH64 only, if there are any concerns. I didn't expect any impact, though, so seemed ok adding unconditionally.

Thanks

Reply:
>- _UID change to 1 is a cosmetic change. The device is currently referred as COM1 everywhere.

There's no such thing as `COM0` so I'd rather keep the `_UID` as 0.

>- _DDN is required as it's referred by some driver code in Windows IoT. There should be no impact to x86 even if this is present.

Sounds good.

Reply:
ACK, the `_UID` part reverted. In case another serial device must be there, it can be `ONE` then.

Thanks